src/category_theory/monoidal/center.lean

```diff
@@ -0,0 +1,86 @@
+/-
+Copyright (c) 2020 Scott Morrison. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Scott Morrison
+-/
+import category_theory.monoidal.braided
+
+/-!
+# Half braidings and the Drinfeld center of a monoidal category
+
+We define `center C` to be pairs `⟨X, b⟩`, where `X : C` and `b` is a half-braiding on `X`.
+
+For now we only define the `category` structure on `center C`.
+Writing down the `monoidal_category` and `braided_category` data is easy enough,
+but verifying the axioms seems intimidating!
+-/
+
+open category_theory
+open category_theory.monoidal_category
+
+universes v v₁ v₂ v₃ u u₁ u₂ u₃
+noncomputable theory
+
+namespace category_theory
+
+variables {C : Type u₁} [category.{v₁} C] [monoidal_category.{v₁} C]
+
+/--
+A half-braiding on `X : C` is a family of isomorphisms `X ⊗ U ≅ U ⊗ X`, natural in `U : C`.
+
+Thinking of `C` as a 2-category with a single `0`-morphism, these are the same as
+natural transformations (in the pseudo- sense)
+of the identity 2-functor on `C`, which send the unique `0`-morphism to `X`.
```

Comment:
```suggestion
Thinking of `C` as a 2-category with a single `0`-morphism, these are the same as natural 
transformations (in the pseudo- sense) of the identity 2-functor on `C`, which send the unique 
`0`-morphism to `X`.
```
I don't mind if you prefer using width 80 instead, but currently this seems a little odd :)

Reply:
Sure! I try to use semantic line breaks when writing markdown/html/latex etc where they don't affect the final formatted version, but I agree the line breaks here make no sense. :-)